elyra/pipeline/airflow/processor_airflow.py

```diff
@@ -280,12 +280,24 @@ def _cc_pipeline(self, pipeline, pipeline_name):
                         continue
 
                     # Get corresponding property's value from parsed pipeline
-                    property_value = operation.component_params.get(component_property.ref)
+                    property_value_dict = operation.component_params.get(component_property.ref)
+                    property_value = property_value_dict[property_value_dict['activeControl']]
+
+                    self.log.debug(f"Property value : {property_value}")
 
                     self.log.debug(f"Processing component parameter '{component_property.name}' "
                                    f"of type '{component_property.data_type}'")
 
-                    if component_property.data_type == "string":
+                    if property_value and not type(property_value) == bool and \
+                            str(property_value)[0] == '{' and str(property_value)[-1] == '}':
+                        pv = json.loads(json.dumps(property_value))
+                        if isinstance(pv, dict) and set(pv.keys()) == {'value', 'option'}:
+                            parent_node_name = self._get_node_name(sorted_operations, pv['value'])
+                            processed_value = "\"{{ ti.xcom_pull(task_ids='" + parent_node_name + "') }}\""
+                            operation.component_params[component_property.ref] = processed_value
+                    elif component_property.data_type == "boolean":
```

Comment:
is `type(property_value) == bool` always true when `component_property.data_type == "boolean"`?  If so, I would move this to the primary `if` statement so that `not type(property_value) == bool` can be removed from the check for the xcom stuff.

Reply:
removed, I think this was older remnant from a commit I forgot to remove, thanks